src/app/routes/ssoRoutes.js

```diff
@@ -423,13 +423,8 @@ module.exports = (app) => {
     }
   });
 
-  app.all('/learner/get/user/sessionId/:userId', (req, res) => {
-      if (req.session.userId === req.params.userId) {
-        res.send({id: getEncyptedQueryParams({userName: req.session.userName})})
-      } else {
-        throw 'unhandled exception while getting sessionID';
-      }
-
+  app.all('/get/user/sessionId', (req, res) => {
```

Comment:
here we can use the .get only instood of .all
and how are we seeing if this is called by a loggedin user only